compass/landice/tests/ismip6_forcing/ocean_basal/process_basal_melt.py

```diff
@@ -0,0 +1,227 @@
+import os
+import shutil
+import subprocess
+import xarray as xr
+from compass.landice.tests.ismip6_forcing.create_mapfile \
+    import build_mapping_file
+from mpas_tools.io import write_netcdf
+from compass.step import Step
+
+
+class ProcessBasalMelt(Step):
```

Comment:
I'm not sure where the issue arises, but every time I run this step I get a different value of gamma0 for a given output file, ranging from 11e3 to >4e5 for `processed_basin_and_coeff_gamma0_DeltaT_quadratic_non_local_median.nc`.

Reply:
Looking back at `case_outputs/landice_ismip6_forcing_ocean_basal.log`, it looks like it is looping through starting at a different file every time:
The first time I ran the step, it started with: 
```
processing the input file coeff_gamma0_DeltaT_quadratic_local_median_PIGL_gamma_calibration.nc
```
The next time, it started with:
```
processing the input file coeff_gamma0_DeltaT_quadratic_local_median.nc
```
The next time:
```processing the input file coeff_gamma0_DeltaT_quadratic_local_95th_percentile.nc```

etc.
So maybe the order of the input files is changed every time, but the order of the output files remains the same, so they are not being written to the correct file?

Reply:
I did find this issue just before my trip to conference in Singapore and tried to look at it today but did not manage to find where and why this mix-up occurs. @xylar - Could you take a look at this part of the code and see if you can spot any source of the error?  Thanks so much!!

Reply:
I think this discussion belongs in #377 rather than here, right?  It doesn't relate to the RACMO step added in this PR.  That being said, I'll look into it in #377

Reply:
Okay, so as I just said to @hollyhan on Slack, I now think this PR should just be closed and the RACMO commits should be included in the branch for #377.  I don't think #377 works without these changes so we just need to include everything there.  Does that sound right?

Reply:
I think I may see what's going wrong with the `ocean_basal` step.  I'll comment on the incorrect code below.

Reply:
This problem has been solved by adding a counter to the file names `combined.nc` and 'remapped.nc` so that the file names are different in the loop going through the source files. For some unknown reasons, `ncremap` failed in interpolating the `gamma0`values when using the same names for the temporary files even when the files were removed at the end of each loop. This solution does not solve the fundamental problem that seems to reside in `ncremap`, but probably the best for making the tool work for now.